cli/utils.go

```diff
@@ -21,13 +21,13 @@ import (
 	"os"
 	"path"
 
-	"github.com/prometheus/alertmanager/client"
+	"github.com/prometheus/alertmanager/api/v2/client/general"
+	"github.com/prometheus/alertmanager/api/v2/models"
+	"github.com/prometheus/alertmanager/config"
```

Comment:
`github.com/prometheus/alertmanager/config` is imported twice (see line below)...

Reply:
Fixed!